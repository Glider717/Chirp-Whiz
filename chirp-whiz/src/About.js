import React from 'react';

function About() {
    return (
        <div>
            <h1>About this Web Application:</h1>
            <p>Chirp Whiz started off as a simple idea for bird-lovers 
                who are stuck in the city or don't have spare time. 

                Please peruse the glossary to get to know the birds on 
                our app and then quiz yourself to see how well you do at 
                identifying our birds! You can check how well you identify 
                each bird in your stats on the User page. You can also view 
                the top players in our leaderboard in the Quiz page.
            </p>
            <br/>
            <h1>Contact Us:</h1>
            <p>Email: lboylan000@citymail.cuny.edu</p>
            <p>If you want to see a bird that we do not have on our  
               website please let us know! It would really help if 
               you can include the name of the bird and links to audio 
               and images of it!</p>
            <br/>
            <h4>Credits:</h4>
            <p>This website was made possible by the United States 
                Geological Survey (USGS) and The Cornell Lab. Our 
                bird songs have been sourced from the USGS. All 
                other bird information and images are from The 
                Cornell Lab's All About Birds website.
            </p>
            <br/>
            <p>Thanks for checking out this site!</p>
            <p>Hangill Chong</p>
            <p>Liam Boylan</p>
            <p>Luis Medina</p>
            <p>Zohora Noor</p>
        </div>
    );
}

export default About;
