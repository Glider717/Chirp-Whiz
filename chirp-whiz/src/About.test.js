import React from 'react';
import About from './About';
import '@testing-library/jest-dom/extend-expect'
import { render, fireEvent, waitForElement } from '@testing-library/react'

it('test About renders', () => {
    const container = render(<About />)    
    expect(container.firstChild).toMatchSnapshot();
});

test("Check if contact us heading is on the page", () => {
    const {getByText} = render(<About />)
    expect(getByText("Contact Us:")).toBeInTheDocument();
});