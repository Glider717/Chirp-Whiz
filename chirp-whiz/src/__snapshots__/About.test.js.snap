// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test About renders 1`] = `undefined`;
